packages/server/lib/browsers/webkit.ts

```diff
@@ -147,5 +147,11 @@ export async function open (browser: Browser, url: string, options: BrowserLaunc
     }
   }
 
+  await utils.executeAfterBrowserLaunch(browser, {
+    get webSocketDebuggerUrl (): never {
+      throw new Error('The `webSocketDebuggerUrl` property is not currently supported in the `after:browser:launch` event handler details argument')
```

Comment:
Why would this throw in Webkit vs logging an error and continuing? Wouldn't this fail for webkit testing?

Reply:
Because any code relying on getting the websocket url would fail with a less useful message later. This makes it clear to the user why their test is failing in webkit.

Reply:
In [writing a test for this](https://github.com/cypress-io/cypress/pull/28180/commits/03ae6a62777b6a1a8863aee7ce7bd2f5a5e59fdf), I realized it's pretty trivial to provide the websocket url instead of throwing an error, so it's now supported in webkit.